### ISSUE TITLE
Vab can't install Android API level

### ISSUE THREAD

**Issue by Tembocs** [vlang/vab#92]
Ubuntu 20.10 x64
V full version: V 0.2.2 5306b27
vab version 0.2.1 9c93ea8

**What did you do?**
Run ```./vab install "platforms;android-30"```

**What did you expect to see?**
Android API level 30 to be installed.

**What did you see instead?**
```
Warning: Failed to find package android-30                                      
[===                                    ] 10% Computing updates...     
```
I have a working internet through which I installed the Android NDK.

**Comment by Larpon** [comment]
@Tembocs can you please post the output of `./vab doctor` - we need more info in order to understand your setup 🙂

**Comment by Tembocs** [comment]
My ```./vab doctor``` output.
```
Android API level "" is not available in SDK.
It can be installed with `vab install "platforms;android-<API LEVEL>"`
Android API level "" is less than the supported level (21).
A vab compatible version can be installed with `vab install "platforms;android-21"`
Android build-tools version  is not available in SDK.
(A vab compatible version can be installed with `vab install "build-tools;24.0.3"`)
vab
	Version 0.2.1 9c93ea8

	Path "/home/username/qnd/v/vab"
Java
	JDK
		Version 15.0.2
		Path "/home/username/Tools/jdk-15.0.2+7"
Android
	ENV
		sdkmanager "/home/username/.cache/v/android/sdk/cmdline-tools/tools/bin/sdkmanager"
		sdkmanager.version "4.0.1"
		Managable: true
	SDK
		Path "/home/username/Android/Sdk"
		Writable true
	NDK
		Version 23.0.7123448
		Path "/home/username/Android/Sdk/ndk"
		Side-by-side true
	Build
		API 
		Build-tools 
	Packaging
		Format apk
Product
	Name "V Test App"
	Package ID "io.v.android"
	Output "v_test_app.apk"
V
	Version 0.2.2 5306b27
	Path "/home/username/qnd/v/v"

	OS: linux, Ubuntu 20.10
	Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz
	CC version: cc (Ubuntu 10.2.0-13ubuntu1) 10.2.0
	
	getwd: /home/username/qnd/v/vab
	vmodules: /home/username/.vmodules
	vroot: /home/username/qnd/v/v
	vexe: /home/username/qnd/v/v/v
	vexe mtime: 2021-02-22 13:20:34
	is vroot writable: true
	is vmodules writable: true
	V full version: V 0.2.2 5306b27
	
	Git version: git version 2.27.0
	Git vroot status: weekly.2021.8-10-g5306b278 (24 commit(s) behind V master)
	.git/config present: true
	thirdparty/tcc status: thirdparty-linux-amd64 7543de81
```

**Comment by Larpon** [comment]
@Tembocs - thanks for the report, it was a big help :slightly_smiling_face: 

It should be fixed in e71a4d8
